massa-versioning/src/versioning.rs

```diff
@@ -899,7 +899,7 @@ impl MipStoreRaw {
 
             let vote_ratio_ = 100.0 * network_version_count / block_count_considered;
 
-            let vote_ratio = Amount::from_mantissa_scale(vote_ratio_.round() as u64, 0);
+            let vote_ratio = Amount::const_init(vote_ratio_.round() as u64, 0);
```

Comment:
Just curious, vote_ration is not const sont const_init will be called at runtime, so why replacing from_mantissa_scale by const_init?

Reply:
Good point. Do you mind replacing all the runtime calls back to from_mantissa_scale to avoid panics ?

Reply:
ok, gonna do that :)

Reply:
Won't change this one, I've added #4099 to track it.